src/__internal__/checkable-input/checkable-input.component.tsx

```diff
@@ -133,7 +133,7 @@ const CheckableInput = React.forwardRef(
       name: id,
       reverse,
       warning,
-      validationIconId,
+      validationId,
```

Comment:
I don't think this would be recognised by `FormField` as it would expect a `validationIconId` prop instead. What do you think about doing this? 

```suggestion
      validationIconId: validationId,
```

Reply:
nice catch, looks like TypeScript compiler did not catch it.
I have fixed that and added a type to that object, so it will not happen in the future. Looks like there was an unnecessary `name` prop that I also removed.